recipes/moreutils/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "moreutils" %}
+{% set version = "0.65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://deb.debian.org/debian/pool/main/m/{{ name }}/{{ name }}_{{ version }}.orig.tar.xz
+  sha256: ba0cfaa1ff6ead2b15c62a67292de66a366f9b815a09697b54677f7e15f5a2b2
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+  run:
+    - perl
+
+test:
+  commands:
+    - echo "foo" | ts
```

Comment:
Should the other tools included be tested as well?

Reply:
There isn't really a test suite for this package, but I added another small test.